package/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "miden-package"
+version = "0.11.0"
+description = "Miden VM package"
+documentation = "https://docs.rs/miden-package/0.11.0"
```

Comment:
Let's increment the versions to v0.12.0.

Reply:
Thanks! Done.